From fe30ea69ff81f99607b0e4002ef9ae12e4694b31 Mon Sep 17 00:00:00 2001
From: Namjae Jeon <namjae.jeon@samsung.com>
Date: Wed, 26 May 2021 16:45:24 +0900
Subject: [PATCH] cifsd: Blank lines aren't necessary after an open brace '{'
Git-commit: fe30ea69ff81f99607b0e4002ef9ae12e4694b31
Patch-mainline: v5.15-rc1
References: bsn#12
Modified-by-SEL: No


Blank lines aren't necessary after an open brace '{'.

Signed-off-by: Namjae Jeon <namjae.jeon@samsung.com>
Signed-off-by: Steve French <stfrench@microsoft.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 fs/cifsd/smb2pdu.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/fs/cifsd/smb2pdu.c b/fs/cifsd/smb2pdu.c
index bdaa69facee5..16290ad710fa 100644
--- a/fs/cifsd/smb2pdu.c
+++ b/fs/cifsd/smb2pdu.c
@@ -2388,7 +2388,6 @@ int smb2_open(struct ksmbd_work *work)
 		rc = -EINVAL;
 		goto err_out1;
 	} else {
-
 		if (req->CreateOptions & FILE_SEQUENTIAL_ONLY_LE &&
 		    req->CreateOptions & FILE_RANDOM_ACCESS_LE)
 			req->CreateOptions = ~(FILE_SEQUENTIAL_ONLY_LE);
-- 
2.33.0

